### PR TITLE
Plumb through the ability to set the storage read cache size.

### LINKED ISSUE #1772: Make LRU size tunable

### PR DESCRIPTION
**Pull request by jefferai**
Fixes #1772

### CODE REVIEW

File: website/source/docs/config/index.html.md [hunk 1]
```diff
@@ -50,6 +50,10 @@ sending a SIGHUP to the server process. These are denoted below.
   "tcp" is currently the only option available. A full reference for the
    inner syntax is below.
 
+* `cache_size` (optional) - A boolean. If set, the size of the read cache used
```
Comment (vishalnayak) [inline website/source/docs/config/index.html.md]: Not a boolean.

**Comment by vishalnayak** [comment]
One comment, other than that LGTM!

File: website/source/docs/config/index.html.md (hunk 1 above)
Comment (jefferai) [inline website/source/docs/config/index.html.md]: Haha!

**Comment by vishalnayak** [comment]
Also, the test case needs to be updated.

**Pull request merged**